ui/app/AppLayouts/Communities/popups/CreateCommunityPopup.qml

```diff
@@ -363,7 +363,14 @@ StatusStackModal {
         StatusScrollView {
             id: generalView
             contentWidth: availableWidth
-            readonly property bool canGoNext: generalViewLayout.isNameValid && generalViewLayout.isDescriptionValid && (root.isDevBuild || (generalViewLayout.isLogoSelected && generalViewLayout.isBannerSelected))
+
+            readonly property var nextAction: () => {
+                generalViewLayout.validate(root.isDevBuild)
+                if (generalViewLayout.isNameValid && generalViewLayout.isDescriptionValid &&
+                    (root.isDevBuild || (generalViewLayout.isLogoSelected && generalViewLayout.isBannerSelected && generalViewLayout.areTagsSelected))) {
```

Comment:
Could we make `validate()` return a bool to say if the needed fields all pass the validation. It would make this code simpler, especially since we duplicate the logic of some fields not needing validation in dev.

Reply:
Yeah, that's a good idea probably, will try

Reply:
Done